docs(process): coc review notes (#540)
nodejs/security-wg: processes/security_team_onboarding.md

Once acceptance has been acquired by group members, the following should take place to setup the new member:

* Confirm that the new member has read, understands, and agrees to uphold the

>   [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities at the Security WG is about handling sensitive matters with open source maintainers and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation WG team and ask for guidance. As a member of the Security WG please also be aware of public statements your make and interactions.

Reviewer: Moderation is a team not a wg.

Reviewer 2: Pushed commit to fix that.  @lirantal hope that's ok.

Author: of course it is! thank you both ;-)

>   [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities at the Security WG is about handling sensitive matters with open source maintainers and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation WG team and ask for guidance. As a member of the Security WG please also be aware of public statements your make and interactions.

Reviewer 3: ```suggestion
  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities in the Security WG is about handling sensitive matters with open source maintainers, and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation team and ask for guidance. As a member of the Security WG please also be aware of public statements you make.
```

>   [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities at the Security WG is about handling sensitive matters with open source maintainers and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation WG team and ask for guidance. As a member of the Security WG please also be aware of public statements your make and interactions.

Reviewer 4: The moderation group can help, but I  think asking on the security-wg slack is a good first step, too. Or technology aside, its good to work together with other people on the sec-wg before going the next step and involving moderation team.

Author: Understand where you're coming from and hopefully it should go without saying that discussing it with friends on the wg slack is a good first step. With that said, in the matters of communicating on public forums the sec wg may not be well trained on it even if they have good intentions - therefore outlining here that the Moderation team is a helpful channel is important to mention in my opinion.

Author: Will resolve the conversation and we can pick it up in a new PR if needed

> * Mentor should ping @jasnell, @gibfahn, @cjihrig or @mhdawson to add the new member to the repository.

Reviewer 3: Unrelated to this PR, but this list should likely be updated. Based on the current membership list, I would drop James and Gibson, and add @sam-github and @ChALkeR (assuming they're OK with it).

Reviewer 2: +1 from me

Author: please confirm @sam-github and @ChALkeR

Author: I'll open a follow-up PR for this
